bundle/manifests/search-v2-operator.clusterserviceversion.yaml

```diff
@@ -92,7 +92,13 @@ spec:
           resources:
           - deployments
           verbs:
-          - '*'          
+          - '*'    
+        - apiGroups: ["*"]
+          resources: ["*"]
+          verbs:
+          - get
+          - list
+          - watch        
```

Comment:
This access seems too broad.

Reply:
This is what we have for search-collector now @jlpadilla

Reply:
ok, I see we combined everything into a single service account.  We'll need to check now this affects the threat model because now all pods have the same access.

Reply:
Yes , we need to define that , single service account vs multiple.